app/uk/gov/hmrc/gform/submission/RoboticsXMLGenerator.scala

```diff
@@ -35,13 +37,23 @@ object RoboticsXMLGenerator {
   ): NodeSeq =
     <gform id={formId.value} dms-id={dmsId} submission-reference={submissionReference.withoutHyphens}>{
       buildObjectStructureXml(structuredForm)
-    }{dateSubmitted(now)}</gform>
+    }{dateSubmitted(now)}{datetimeSubmitted(now)}</gform>
 
   private val dateSubmittedFormater = DateTimeFormatter.ofPattern("dd/MM/yyyy").withZone(ZoneId.of("Europe/London"))
 
   private def dateSubmitted(now: Instant): NodeSeq =
     <dateSubmitted>{dateSubmittedFormater.format(now)}</dateSubmitted>
 
+  private val datetimeSubmittedFormater = new DateTimeFormatterBuilder()
+    .append(ISO_LOCAL_DATE_TIME)
+    .optionalStart()
+    .appendOffsetId()
+    .optionalStart()
+    .toFormatter()
```

Comment:
It seems there is predefined formatter:
```suggestion
  private val datetimeSubmittedFormater = DateTimeFormatter.ISO_OFFSET_DATE_TIME
```

Reply:
done, thanks!